pymeasure/instruments/ni/virtualbench.py

```diff
@@ -128,6 +128,7 @@ def shutdown(self):
         log.info("Shutting down %s" % self.name)
         self.vb.release()
         self.isShutdown = True
+        super().shutdown()
```

Comment:
This instrument surprisingly (to me) does not inherit from `Instrument` class, so super is not needed and it will fail.

Reply:
Yes, the VirtualBench is a separate beast. We should probably clean this up at some point and derive from `Instrument`. I don't know off the cuff how "special" it is, though.